src/MeCab.jl

```diff
@@ -76,16 +76,12 @@ function create_node(raw::MecabRawNode)
 end
 
 function create_surface(raw::MecabRawNode)
-  start_idx = convert(Int64, raw.surface)
-  _nextraw = unsafe_load(raw.next)
-  next_idx = convert(Int64, _nextraw.surface)
-
   local surface::String
   surface = try
-      unsafe_string(raw.surface, next_idx - start_idx)
-    catch
-      unsafe_string(raw.surface)
-    end
+     unsafe_string(raw.surface, raw.length)
```

Comment:
Ah ha, you reverted this code.
https://github.com/chezou/MeCab.jl/pull/16/files#diff-551458bfb5979915bb7758c966cae0d7R86

Reply:
yes, this part was slow. I eliminated this calculation and now use libmecab's info instead.